tests/test_state_integrator.cpp

```diff
@@ -49,17 +51,35 @@ BOOST_AUTO_TEST_CASE(test_state_integrator) {
 
   unsigned int debug_mode = 2;
 
-  std::string robot_description;
-  ifstream urdfFile;
-  std::string filename = "/opt/openrobots/share/simple_humanoid_description/urdf/simple_humanoid.urdf";
-  urdfFile.open(filename.c_str());
-  if (!urdfFile.is_open()) {
-    std::cerr << "Unable to open " << filename << std::endl;
-    BOOST_CHECK(false);
-    return;
+  // Get environment variable CMAKE_PREFIX_PATH
+  const string s_cmake_prefix_path = getenv( "CMAKE_PREFIX_PATH" );
+
+  // Read the various paths
+  vector<string> paths;
+  boost::split(paths, s_cmake_prefix_path, boost::is_any_of(":;"));
+
+  // Search simple_humanoid.urdf
+  string filename="";
+  for (auto test_path : paths)
+  {
+    filename = test_path +
+        string("/share/simple_humanoid_description/urdf/simple_humanoid.urdf");
+    if ( boost::filesystem::exists(filename))
+      break;
   }
-  stringstream strStream;
+
+  // If not found fails
+  if (filename.size()==0)
+  {
+    cerr << "Unable to find simple_humanoid.urdf" << endl;
```

Comment:
```suggestion
    cerr << "Unable to find simple_humanoid_description/urdf/simple_humanoid.urdf in CMAKE_PREFIX_PATH" << endl;
```

Reply:
Thanks Joseph I've updated the message :)